### ISSUE TITLE
BUG: OSX: v3.1 was released with missing executable permission of main executable

### ISSUE THREAD

**Issue by fzwoch** [ezQuake/ezquake-source#310]
From memory - when unzipping the full ezQuake bundle the main executable `ezquake-darwin-x86_64` was missing the execute permission (`+x`). 

It could be due to the original release script bundles the minimal application. Perhaps it was repackaged to include more configs and data files. In that process the executable bit may have been lost.

Currently the user clicks the application and it will just do nothing but return as it was never run. This is on top of the issues that we don't have a key for the application itself now are we known as a trusted developer source. So the user has to explicitly allow to run the application via the security panel - which btw gets worse with every major release of macOS.

This makes a cumbersome experience even worse.

**Comment by kuidow** [comment]
Hi, 

Can someone help with this?

I'm on OSX 10.12 and it's impossible to install the 3.1 :(

Thank you!!!

**Comment by fzwoch** [comment]
I think the easiest is it to install nQuake (https://nquake.com). I don't think it suffers from this bug - I'm not sure though.

Else try opening a Terminal and then something like this:

```
chmod u+x ezQuake.app/Contents/MacOS/ezquake
chmod u+x ezQuake.app/Contents/MacOS/ezquake-darwin-x86_64
```

**Comment by yoavapi** [comment]
not working

**Comment by fzwoch** [comment]
That's not a lot of details. Also check version 3.2 where this should have been fixed.

Also note that running from the downloads folder usually does not work on newer macos releases.

**Comment by yoavapi** [comment]
Hi,

I'm trying to run this from the Applications default folder
Are you referring to the PAK file location?

Thanks!

**Comment by meag** [comment]
Closing: this should have been fixed in 3.2 (my script to create builds is now adding to existing @fzwoch's .zip, rather than extracting and then rezipping like a llama windows user)
